app/src/main/java/com/yalta/activities/DriverMainActivity.kt

```diff
@@ -9,23 +9,28 @@ import com.yalta.activities.fragments.BlankFragment
 import com.yalta.activities.fragments.DriverBrowseFragment
 import com.yalta.activities.fragments.MapFragment
 import com.yalta.activities.fragments.ProfileFragment
+import com.yalta.di.YaltaApplication
 import com.yalta.utils.ViewUtils.grantedLocationPermission
+import javax.inject.Inject
 
 class DriverMainActivity : AppCompatActivity() {
-    private lateinit var browseFragment: Fragment
-    private lateinit var mapFragment: Fragment
-    private lateinit var profileFragment: Fragment
-    private lateinit var currentFragment: Fragment
+
+    @Inject
+    lateinit var browseFragment: DriverBrowseFragment
+    @Inject
+    lateinit var profileFragment: ProfileFragment
+    lateinit var mapFragment: Fragment
+    lateinit var currentFragment: Fragment
```

Comment:
`private` is gone

Reply:
![image](https://user-images.githubusercontent.com/2794213/103351215-8e2f5480-4ab3-11eb-9afc-23625a7f55b0.png)
Private Isgone